Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -1198,6 +1207,34 @@ static Class _WebMessageDocumentClass()
     return _WebMessageDocumentClass;
 }
 
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
+BOOL HTMLConverter::_addMultiRepresentationHEICAttachmentForImageElement(HTMLImageElement& element)
+{
+    RefPtr image = element.image();
+    if (!image)
+        return NO;
+
+    WebMultiRepresentationHEICAttachment *attachment = image->adapter().multiRepresentationHEIC();
+    if (!attachment)
+        return NO;
+
+    NSUInteger textLength = [_attrStr length];
+
+    RetainPtr<NSString> string = adoptNS([[NSString alloc] initWithFormat:@"%C", static_cast<unichar>(NSAttachmentCharacter)]);
```

Comment:
what’s the difference between a unichar and a unsigned char?

Reply:
It's an `unsigned short`, and is specifically used for [UTF-16 code units](https://developer.apple.com/documentation/foundation/unichar?language=objc).

This initializer is already used elsewhere in the file.